knative-operator/pkg/controller/dashboard/dashboard.go

```diff
@@ -91,3 +86,11 @@ func getAnnotationsFromInstance(instance operatorv1alpha1.KComponent) mf.Transfo
 	}
 	return nil
 }
+
+func getDashboardsPath(subPath string) string {
+	path := os.Getenv(DashboardsManifestPathEnvVar)
+	if path == "" {
+		return "deploy/resources/dashboards" + "/" + subPath
+	}
```

Comment:
Is this used in tests? If not: Can we drop it? I'm a bit cautious about defaults that should never actually be in-use :joy:

Reply:
Reading along it **seems** like the tests are actually using the env vars and this default is the "prod" path. I wonder if we should drop the default and use an env var in prod as well, just to have uniformity (and to avoid silent defaults). WDYT?